packages/desktop/views/dashboard/settings/views/advanced/advanced-settings.constant.js

```diff
@@ -1,8 +1,8 @@
-import { BalanceFinder, HiddenAccounts, DeveloperToggle } from '.'
+import { BalanceFinder, HiddenWallets, DeveloperToggle } from '.'
```

Comment:
If `HiddenAccounts.svelte` is no longer used, it can be deleted, right?

Reply:
damn conflicts...